src/main/java/com/redislabs/redistimeseries/MultiRangeParams.java

```diff
@@ -41,6 +43,11 @@ public MultiRangeParams withLabels(boolean withLabels) {
     return this;
   }
 
+  public MultiRangeParams groupByReduce(String group, String reduce) {
+    this.groupByReduce = new String[] {group, reduce};
```

Comment:
@sazzad16 what is the advantages of setting these two as String[] and not as two fields? It feels like a waste of allocation